src/index.js

```diff
@@ -43,21 +44,15 @@ class IpfsRepo {
   constructor (repoPath, options) {
     assert.equal(typeof repoPath, 'string', 'missing repoPath')
 
-    if (options == null) {
-      options = require('./default-options')
```

Comment:
Changing this means that any module bundle will include the default options in the browser always. Not sure this could be an issue or not